test/python/circuit/test_parameter_expression.py

```diff
@@ -15,16 +15,13 @@
 import cmath
 import math
 import unittest
-
-from test import combine
-from test import QiskitTestCase
+from test import QiskitTestCase, combine
```

Comment:
I suspect it may be quicker/easier to review this PR if the only changes to this file are addition of tests and any new imports that might be necessary. The unrelated changes should be kept to a minimum to enable speedy review.